### PR TITLE
Fix deb package build

### PR DESCRIPTION
**Pull request by legleux**
## High Level Overview of Change
Use GCC-11 and update GCC Conan profile

### Context of Change
This is a quick band aid to get this build fixed.
A goal of Q3 is to remove all this package building code.

Also renames debug packages `ddeb` --> `deb` to account for an Artifactory bug where the debug symbol packages with `.ddeb` extensions are not indexed and thus aren't visible in apt clients.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: Builds/containers/packaging/dpkg/debian/rules [hunk 1]
```diff
@@ -38,12 +51,14 @@ override_dh_auto_configure:
...
 
 	conan install . \
+		--profile gcc \
 		--install-folder bld/rippled-reporting \
 		--build missing \
 		--build boost \
 		--build sqlite3 \
 		--build libuv \
 		--settings build_type=Release \
+		--settings compiler.cppstd=17 \
```
Comment (thejohnfreeman) [inline Builds/containers/packaging/dpkg/debian/rules:61]: We can't build non-reporting mode _without_ C++20, and we can't build reporting mode _with_ C++20? Is that right?

Comment (legleux) [reply Builds/containers/packaging/dpkg/debian/rules:61]: It seems to be the case. IIRC, there was an issue building Cassandra.

Comment (thejohnfreeman) [reply Builds/containers/packaging/dpkg/debian/rules:61]: Yeah, that's the issue I'm having:

> C++20 changes std::memory_order from an `enum` to an `enum class`. Datastax supplies a C++ driver for Cassandra, a dependency of reporting mode. it [defines](https://github.com/datastax/cpp-driver/blob/84389657d4504604c5d05b87769ce3ce983a2296/src/atomic/atomic_std.hpp#L24-L31) an `enum MemoryOrder` whose values are copied from `std::memory_order`, but using values from an `enum class` is not allowed, causing a compile error: https://godbolt.org/z/dG943nvT7

**Review by thejohnfreeman — APPROVED** [review]

**Review by godexsoft — APPROVED** [review]
LGTM 👍

**Pull request merged**